l3build-check.lua

```diff
@@ -952,6 +962,9 @@ function check(names)
     end
     if errorlevel ~= 0 then
       checkdiff()
+      if options["show-save-cmds"] then
+        showsavecommands(failurelist)
```

Comment:
name consistency
show_save_cmds

Reply:
Consistency favors `show-save-cmds`: It is a l3build option. There are no existing l3build options with underscores and multiple existing options with hyphens (`--dry-run`, `--halt-on-error`, and `--show-log-on-error`)

Reply:
The suggestion was targeting the last line.
compare
```
if options["show-save-cmds"] then
  showsavecommands(failurelist)
```
with
```
if options["show-save-cmds"] then
  show_save_cmds(failure_list)
```

Reply:
Then take the similar comparison to `showfailedlog` or `showfaileddiff`. I am aware that you don't like the coding style in `l3build` and I agree that it isn't particularly consistent. But since changing the existing functions comes with compatibility and stability concerns, we have to accept it and try to name new functions as consistently as possible.